src/components/footer.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Footer = () => (
+  <></>
+);
+
+export default Footer;
```

Comment:
This footer component is empty.

Please either implement this component or remove it in order not to have an unused component.